sdk/generic-sdk-core/generic-core-http-okhttp/src/main/java/com/generic/core/http/okhttp/implementation/OkHttpBufferedResponse.java

```diff
@@ -10,18 +10,23 @@
 /**
  * An OkHttp response where the response body has been buffered into memory.
  */
-public final class OkHttpBufferedResponse extends OkHttpResponseBase {
-    private final byte[] body;
+public final class OkHttpBufferedResponse extends OkHttpResponseBase<byte[]> {
```

Comment:
This change makes me think this should be changed to `OkHttpResponseBase<BinaryData>` instead of `byte[]`